AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1360,6 +1361,7 @@ quotev.com#$#body #res_area ::selection { background-color: #3297FD!important; }
 !
 !--- JS ----
 !
+tistory.com#%#AG_onLoad(function() { setTimeout(function() { var GStyle = document.createElement('style'); document.body.appendChild(GStyle); GStyle.type = "text/css"; GStyle.appendChild(document.createTextNode(':not(input):not(textarea):not(code):not(blockquote):not(pre) { user-select: auto!important; -webkit-user-select: auto!important; -webkit-touch-callout: auto!important; }')) }, 800); });
```

Comment:
I told you earlier, we should avoid using JS. Check these rules.
```
tistory.com#$#html * { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
tistory.com#$#html { -webkit-touch-callout: default!important; -moz-user-select: auto!important; -webkit-user-select: auto!important; -ms-user-select: auto!important; -o-user-select: auto!important; user-select: auto!important; -khtml-user-select: auto!important; }
```

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/92116/commits/e3e83ac9c1d51fa0654ad712f4126fc1b3fa6c4f

Reply:
It doesn't work with `body`, I gave you rules with `html`.